fog/distribution/src/main.rs

```diff
@@ -355,23 +363,46 @@ fn worker_thread_entry(
 
         // Send to the next fog account
         let to_account = &fog_accounts[txs_created % fog_accounts.len()];
-
-        // Got our inputs, construct transaction.
-        let tx = build_tx(
-            &pending_spendable_txouts,
-            to_account,
-            &config,
-            &ledger_db,
-            fog_resolver.clone(),
-            &logger,
+        let env = Arc::new(
+            grpcio::EnvBuilder::new()
+                .name_prefix("FogPubkeyResolver-RPC".to_string())
+                .build(),
```

Comment:
this `grpcio::EnvBuilder` object contains an entire thread-pool

ideally, there would only be one of these for the whole app, and we pass it to the worker thread, and the `build_fog_resolver` function

i think it would be okay for the `build_fog_resolver` function to also build the `GrpcFogReportConnection`